Screens/LanguageScreen.js

```diff
@@ -7,7 +7,6 @@ import { setLanguage } from 'redux-i18n';
 class LanguageScreen extends Component {
     setLang = (code) => {
         this.props.storeSettings(this.props.deviceId, { languageCode: code });
-        this.props.setLanguage(code);
```

Comment:
Why did you remove this line?

Reply:
We don`t need it now. We call that function inside storeSettings.